modules/dcache-vehicles/src/test/java/diskCacheV111/vehicles/JobInfoTests.java

```diff
@@ -0,0 +1,74 @@
+package diskCacheV111.vehicles;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.concurrent.*;
+
+/**
+ * Created by alenaschemmert on 03.08.17.
+ */
+
+public class JobInfoTests {
+
+    private JobInfo jobInfo;
+    private OldJobInfo oldJobInfo;
+    private String ref;
+
+    @Before
+    public void setUp() {
+        jobInfo = new JobInfo(System.currentTimeMillis(), System.currentTimeMillis() + 1000, "working", 1, "waldo", 2407);
```

Comment:
Ideally, unit-tests should be repeatable and always give the same output.  This is not always possible, but is certainly something to strive for.

By using `System.currentTimeMillis`, the patch introduces an element of randomness.  For example, suppose there is some subtle bug that only shows up with the last digit is a 0 the test would then fail (roughly) 1 time in 10.

Wherever possible, it's better to use constant values.  When a bug is discovered (e.g., the last digit is a 0 bug) an additional test is adding that targets exactly the conditions that trigger the bug.  That way testing is precise and predictable.